app/templates/index.hbs

```diff
@@ -71,12 +71,14 @@
       {{/layout.blowout}}
 
       {{#layout.cards as |row|}}
-        {{row.brick item=model.wqxrHome.featuredItems.[3]}}
-        {{row.brick item=model.wqxrHome.featuredItems.[4]}}
-        {{row.brick item=model.wqxrHome.featuredItems.[5]}}
+        {{#each (slice model.wqxrHome.featuredItems 3 6) as |item|}}
```

Comment:
Won't this render the scaffolding of each item even it it's empty, like before?

Reply:
no, since slice will create an empty array if 3,4, and 5 don't exist, and then the each block won't render anything since the array is empty.

Reply:
ok, great.